src/main/resources/swagger/api-docs.yaml

```diff
@@ -4091,6 +4149,33 @@ components:
           type: boolean
           description: based on the user's currently accepted terms of service version and when the terms of service were last changed,
             should the user still be permitted to use the system?
+    UserTermsOfServiceHistory:
+      required:
+        - history
+      type: object
+      description: a user's terms of service action history
+      properties:
+        latestAcceptedVersion:
```

Comment:
this variable name doesn't really indicate that it would be an array. Maybe something like `termsOfServiceEvents`?

Reply:
oh woops yeah this should be history or something